wifiphisher.py

```diff
@@ -775,6 +797,7 @@ def get_hostapd():
     # Main loop.
     try:
         while 1:
+            time.sleep(5)
```

Comment:
This may makes sense but it is not relevant to the PR's purpose. Do you mind making this a different PR?